pkg/migrations/seq_num.go

```diff
@@ -12,7 +13,7 @@ func CurrentDBVersionSeqNum() int {
 	if env.PostgresDatastoreEnabled.BooleanSetting() {
 		return internal.CurrentDBVersionSeqNum
 	}
-	return internal.LastRocksDBVersionSeqNum
+	return mathutil.MinInt(internal.LastRocksDBVersionSeqNum, internal.CurrentDBVersionSeqNum)
```

Comment:
If we are going to do it this way, do we still need the `LastRocksDBVersionSeqNum()` below?  I'm a little fuzzy on how this makes it more clear what is happening.

Reply:
Yes, we will still need it. Before we change, we are still using these codes in mockCentral which may have PostgresDatastoreEnabled off.

Reply:
Got it, mostly for all the test scenarios.